deploy/helm/prometheus-overrides.yaml

```diff
@@ -6,29 +6,43 @@ grafana:
 prometheus:
   additionalServiceMonitors:
     - name: fluentd
-      additionalLabels: 
+      additionalLabels:
         app: sumologic
-        release: prometheus-operator
+        release: collection
       endpoints:
       - port: metrics
       namespaceSelector:
         matchNames:
         - sumologic
       selector:
-          matchLabels:
-            app: sumologic
+        matchLabels:
+          app: fluentd
     - name: fluentd-events
-      additionalLabels: 
+      additionalLabels:
         app: sumologic-events
-        release: prometheus-operator
+        release: collection
       endpoints:
-      - port: metrics
+        - port: metrics
       namespaceSelector:
         matchNames:
-        - sumologic
+          - sumologic
+      selector:
+        matchLabels:
+          app: fluentd-events
+    - name: collection-fluent-bit
```

Comment:
@frankreno this file is generated from `values.yaml`. is this `name: collection-fluent-bit` correct for the `prometheus-overrides.yaml` file?